src/routes/ViewAllLogs/ViewAllLogsFilters.js

```diff
@@ -125,6 +125,7 @@ const ViewAllLogsFilters = ({
               dataOptions={getJobProfileOptions}
               value={activeFilters[FILTERS.JOB_PROFILE] ? activeFilters[FILTERS.JOB_PROFILE][0] : ''}
               onChange={onChangeSelectionFilter(onChange, FILTERS.JOB_PROFILE)}
+              placeholder={<FormattedMessage id="ui-data-import.filter.chooseJobProfile" />}
```

Comment:
The `placeholder` prop has a string type, consider wrapping `<Selection>` in `<FormattedMessage>`
https://github.com/folio-org/stripes-components/blob/59fa5ff492972fd1f9d1b8548a30fc33fed68f71/lib/Selection/SingleSelect.js#L49